requirements.txt

```diff
@@ -11,11 +11,12 @@ boto3
 botocore
 moto
 requests 
-setuptools
+setuptools<60.0
```

Comment:
@JiaoMaWHU Could you please clarify why this is needed? Limiting the version of setuptools has created warnings from the Dependabot.